airflow/security/kerberos.py

```diff
@@ -108,18 +108,18 @@ def renew_from_kt(principal: Optional[str], keytab: str, exit_on_fail: bool = Tr
                 return subp.returncode
 
     global NEED_KRB181_WORKAROUND
-    if NEED_KRB181_WORKAROUND is None:
-        NEED_KRB181_WORKAROUND = detect_conf_var()
-    if NEED_KRB181_WORKAROUND:
-        # (From: HUE-640). Kerberos clock have seconds level granularity. Make sure we
-        # renew the ticket after the initial valid time.
-        time.sleep(1.5)
-        ret = perform_krb181_workaround(principal)
-        if exit_on_fail and ret != 0:
-            sys.exit(ret)
-        else:
-            return ret
-    return 0
+    # if NEED_KRB181_WORKAROUND is None:
+    #     NEED_KRB181_WORKAROUND = detect_conf_var()
+    # if NEED_KRB181_WORKAROUND:
+    #     # (From: HUE-640). Kerberos clock have seconds level granularity. Make sure we
+    #     # renew the ticket after the initial valid time.
+    #     time.sleep(1.5)
+    #     ret = perform_krb181_workaround(principal)
+    #     if exit_on_fail and ret != 0:
+    #         sys.exit(ret)
+    #     else:
+    #         return ret
+    # return 0
```

Comment:
This patch was supposed to be added after EWT-1526. Based on that, we do not need this patch

Reply:
Without this change the airflow deployment is failing with the following stack trace:
`Starting pex hydration process...Error
Traceback (most recent call last):
  File "/Users/amalviya/.pex/code/9fb7fa1719cf6a5f5714edac724572d33f77e8c3/twitter/ml/common/kubernetes/lib.py", line 1189, in _kubectl_exec
    stderr=subprocess.STDOUT if capture_stderr else subprocess.PIPE,
  File "/opt/ee/python/3.7/lib/python3.7/subprocess.py", line 411, in check_output
    **kwargs).stdout
  File "/opt/ee/python/3.7/lib/python3.7/subprocess.py", line 512, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['kubectl', '--context=qus1-c1', '-n', 'etl-workflow-users', 'exec', 'airflow-pkg-amalviya-devel-0', '-c', 'airflow-pkg-amalviya-devel', '--', 'sh', '/shared/airflow_package_tmp/bin/gke_devel_pex_hydration.sh']' returned non-zero exit status 1.`

Reply:
Are you testing on staging/prod env? Because only that would use kerberos and 2.2.4 ran fine without this.

Reply:
The devel deployment itself is failing without this.